third_party/txt/src/txt/paragraph_style.h

```diff
@@ -58,6 +66,7 @@ enum TextHeightBehavior {
   kDisableFirstAscent = 0x1,
   kDisableLastDescent = 0x2,
   kDisableAll = 0x1 | 0x2,
+  kHalfLeading = 0x1 << 2,
```

Comment:
I would prefer consistent naming in the engine and the framework. This was called `even` in framework. Two names for the same property makes this much more confusing to read and maintain.